react-app/src/components/ResultPage.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+// import ResultHeader from './ResultHeader';
+import PatientData from './PatientData';
+import ResultSidebar from './ResultSidebar';
+
+import '../stylesheets/ResultPage.scss';
+
+function ResultPage(props) {
+  const [patientID, setPatientID] = useState(-1);
```

Comment:
Per comment in `PatientData.js`, let's initialize this state variable to null instead of -1. That makes things a little cleaner.

```suggestion
  const [patientID, setPatientID] = useState(null);
```